insights/parsers/softnet_stat.py

```diff
@@ -8,42 +8,58 @@
+    02883c47 00007e2e 00000000 00000000 00000000 00000000 00000000 00000000 00000000 02280d49 00000000 00000000 00000003
+    01a35c9d 0002db94 00000001 00000000 00000000 00000000 00000000 00000000 00000000 008ee93a 00000000 00000000 00000004
 
     Column-01: packet_process: Packet processed by each CPU.
     Column-02: packet_drop: Packets dropped.
     Column-03: time_squeeze: net_rx_action.
-    Column-08: cpu_collision: collision occur while obtaining device lock while transmitting.
-    Column-09: received_rps: number of times cpu woken up received_rps.
-    Column-10: flow_limit_count: number of times reached flow limit count.
+    Column-09: cpu_collision: collision occur while obtaining device lock while transmitting.
+    Column-10: received_rps: number of times cpu woken up received_rps.
+    Column-11: flow_limit_count: number of times reached flow limit count.
+    Column-12: softnet_backlog_len: Backlog status
+    Column-13: index: core id owning this softnet_data
+
+For the proc file format details,
+See:
+* Linux 2.6.23 https://elixir.bootlin.com/linux/v2.6.23/source/net/core/dev.c#L2343
+* * example : "6dcad223 00000000 00000001 00000000 00000000 00000000 00000000 00000000 00000000"
+* Linux 4.17 https://elixir.bootlin.com/linux/v4.17/source/net/core/net-procfs.c#L162
+* * example : "00022be3 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000"
+* Linux 5.10 https://elixir.bootlin.com/linux/v5.10/source/net/core/net-procfs.c#L172
+* * example : "00358fe3 00006283 00000000 00000000 00000000 00000000 00000000 00000000 00000000 000855fc 00000000 00000000 00000000"
+* https://elixir.bootlin.com/linux/v4.17/source/include/linux/netdevice.h#L2810
+* https://elixir.bootlin.com/linux/v5.10/source/include/linux/netdevice.h#L3181
 
 .. Note::
             There is minimal documentation about these fields in the file, columns are not labeled and
-            could change between the kernel releases. This format is compatible for RHEL-6
-            and RHEL-7 releases. Also it is unlikely that the positions of those values will
+            could change between the kernel releases. This format is compatible for RHEL-6,
+            RHEL-7, RHEL-8 and RHEL-9 releases. Also it is unlikely that the positions of those values will
             change in short term.
 
 Examples:
     >>> SOFTNET_STATS = '''
-    ... 00008e78 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-    ... 000040ee 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-    ... 0001608c 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-    ... 0000372f 00000001 00000000 00000000 00000000 00000000 00000000 00000000 00000000 00000000
-    ...'''.strip()
+    ... 00358fe3 00006283 00000000 00000000 00000000 00000000 00000000 00000000 00000000 000855fc 00000000 00000000 00000000
```

Comment:
If you're going to update the examples, could you please add in a doc test and update the examples to work properly? The below commit is an example of me adding a doc test to existing parser for reference.

https://github.com/RedHatInsights/insights-core/commit/cbf6b7bdf8ceb0712553d2165f2ff860e0c849e1

Reply:
@remijouannet It doesn't look like you've added a doctest yet, was this marked resolved by accident?

Reply:
sorry i forgot to add a unitest 

```
bin/pytest -v insights/tests/parsers/test_softnet_stat.py
=================================================================== test session starts ====================================================================
platform darwin -- Python 3.9.12, pytest-7.1.3, pluggy-1.0.0 -- /Users/remi.jouannet/git/insights-core/bin/python3.9
cachedir: .pytest_cache
rootdir: /Users/remi.jouannet/git/insights-core, configfile: setup.cfg
plugins: cov-4.0.0
collected 2 items

insights/tests/parsers/test_softnet_stat.py::test_softnet_stat PASSED                                                                                [ 50%]
insights/tests/parsers/test_softnet_stat.py::test_softnet_stat_doc_examples PASSED
```
better ?

Reply:
Yep that's great thank you.